README.md

```diff
@@ -24,8 +24,11 @@ Building The Provider
 Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-rancher2`
```

Comment:
Seems that this line also has the typo. Could you please modify it??

```
-Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-rancher2`
+Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-rke`
```

Reply:
good catch.